xtable-api/src/main/java/org/apache/xtable/model/storage/InternalDataFile.java

```diff
@@ -33,17 +38,18 @@
  *
  * @since 0.1
  */
-@Builder(toBuilder = true)
-@Value
-public class InternalDataFile {
-  // physical path of the file (absolute with scheme)
-  @NonNull String physicalPath;
+@SuperBuilder(toBuilder = true)
+@FieldDefaults(makeFinal = true, level = lombok.AccessLevel.PRIVATE)
+@ToString(callSuper = true)
+@EqualsAndHashCode(callSuper = true)
+@Accessors(fluent = true)
```

Comment:
What is the reason for switching to fluent accessors instead of the get prefix?

Reply:
I find the fluent accessors less verbose, which makes the code easier to read. In the projects I am involved in, it is a preferred pattern, so I used it here. However, after creating the PR, I realized it makes our API inconsistent. I'm supportive of holding off this change for now and discussing it. If the community likes it, we can apply the change in one PR for all APIs. WDYT?

Reply:
Let's keep with the current patterns for now and keep the style changes separate from the deletion vector changes. This is already a breaking change so I would prefer we aim to keep the surface area as small as possible right now so users can avoid having to update so many places in their code.